lighty-modules/lighty-jetty-server/src/main/java/io/lighty/server/Http2LightyServerBuilder.java

```diff
@@ -0,0 +1,59 @@
+/*
+   Copyright (c) 2022 PANTHEON.tech s.r.o. All Rights Reserved.
+
+   This program and the accompanying materials are made available under the
+   terms of the Eclipse Public License v1.0 which accompanies this distribution,
+   and is available at https://www.eclipse.org/legal/epl-v10.html
+ */
+package io.lighty.server;
+
+import io.lighty.server.config.SecurityConfig;
+import java.net.InetSocketAddress;
+import org.eclipse.jetty.alpn.server.ALPNServerConnectionFactory;
+import org.eclipse.jetty.http.HttpVersion;
+import org.eclipse.jetty.http2.server.HTTP2ServerConnectionFactory;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.HttpConnectionFactory;
+import org.eclipse.jetty.server.SecureRequestCustomizer;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+
+public class Http2LightyServerBuilder  extends LightyServerBuilder {
+
+    private final SecurityConfig securityConfig;
+
+    public Http2LightyServerBuilder(final InetSocketAddress inetSocketAddress, final SecurityConfig securityConfig) {
+        super(inetSocketAddress);
+        this.server = new Server();
```

Comment:
We are using `build()` to create an instance of `Server`. Remove `this.server = new Server();` form here.
This class does not look like builder at all - everything is set in constructor - maybe remove constructor and add methods `withAddress(InetSocketAddress)` and `withSecurityConfig(SecurityConfig)`.

Reply:
OK, go with constructor like in parent. But d not create instance in constructor. And yes, you can add `withSecurityConfig(SecurityConfig)`.

Reply:
Done